File: trip-service-java/src/main/java/com/cawadev/softwarecraft/katas/tripservice/exception/CollaboratorCallException.java
package com.cawadev.softwarecraft.katas.tripservice.exception;

import java.io.Serial;

public class CollaboratorCallException extends RuntimeException {

	@Serial
	private static final long serialVersionUID = -4584041339906109902L;

	public CollaboratorCallException(String message) {
		super(message);
	}
}

File: trip-service-kotlin/src/main/kotlin/com/cawadev/softwarecraft/katas/tripservice/exception/CollaboratorCallException.kt
package com.cawadev.softwarecraft.katas.tripservice.exception

class CollaboratorCallException(message: String?, cause: Throwable?) :
    RuntimeException(message, cause) {

    constructor(message: String) : this(message, null)

    constructor(cause: Throwable) : this(null, cause)

    companion object {
        private const val serialVersionUID: Long = -4584041339906109902L
    }
}
